gci/task.py

```diff
@@ -81,12 +84,49 @@ def beginner_tasks(tasks):
 
 def strip_mentors(tasks):
     for task in tasks.values():
-        del task['mentors']
+        if task.__contains__('mentors'):
+            del task['mentors']
 
 
 def cleanse_tasks(tasks):
+    tokens = {
+        'GH_TOKEN': os.environ.get('GH_TOKEN'),
+    }
+
+    for task in tasks.values():
+        if task['max_instances'] == 1 \
```

Comment:
Isn't the explicit use of line continuation discouraged?

Reply:
yes, will make it correct :+1: